net/edf/register.go

```diff
@@ -238,7 +238,12 @@ func registerType(tov reflect.Type) error {
 
 		nf := tov.NumField()
 		for i := 0; i < nf; i++ {
-			ft := tov.Field(i).Type
+			f := tov.Field(i)
+			ft := f.Type
+
+			if f.IsExported() == false {
```

Comment:
```suggestion
			if !f.IsExported(){
```
This is nit

Reply:
I personally prefer to use more readable way of such expressions.